Assets/ScaleMonk Ads/Source/UserType.cs

```diff
@@ -0,0 +1,15 @@
+namespace ScaleMonk.Ads
+{
+    public enum UserType 
+    {
+        NON_PAYING_USER, PAYING_USER
```

Comment:
@marinoluck the convention in C# for Enum types is UpperCamelCase.

Reply:
👍